src/libraries/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -730,7 +739,15 @@ public void IdFormat_W3CInvalidVersionFF()
         [Fact]
         public void IdFormat_W3CWhenTraceIdAndSpanIdProvided()
         {
-            Activity activity = new Activity("activity3");
+            using (var a = new Activity("foo"))
```

Comment:
Can we make a new test method for this code or move it to the other SetParentId test? The test method name sounds pretty accurate for the pre-existing code in this test but not for this new part.

Reply:
I'll move it to the other test method.